examples/random_walk.py

```diff
@@ -0,0 +1,22 @@
+import time
+import random
+
+from ai2thor_wrapper.envs import ThorWrapperEnv
+
+if __name__ == '__main__':
+    # Random agent example with wrapper
+    max_episode_length = 10000
+    env = ThorWrapperEnv(max_episode_length=max_episode_length)
+    for episode in range(20):
+        start = time.time()
+        s = env.reset()
```

Comment:
initial_state, state_ini or sth similar would be preferred to "s". In general I would avoid single letter variables